sdk/keyvault/keyvault-keys/package.json

```diff
@@ -96,7 +96,6 @@
     "mocha": "^5.2.0",
     "nock": "^10.0.6",
     "prettier": "^1.16.4",
-    "query-string": "^6.5.0",
```

Comment:
Not using ?

Reply:
Our recorder.ts doesn't include none of the browser things, to reduce the surface of complexity in it. We'll bring that back as we make sure keyvault can work in the browser.